components/style/values/specified/pointing.rs

```diff
@@ -0,0 +1,109 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+//! Specified values for Pointing properties.
+//!
+//! https://drafts.csswg.org/css-ui/#pointing-keyboard
+
+use cssparser::Parser;
+use parser::{Parse, ParserContext};
+use selectors::parser::SelectorParseErrorKind;
+use style_traits::ParseError;
+use style_traits::cursor::CursorKind;
+
+/// The specified value for the `cursor` property.
+///
+/// https://drafts.csswg.org/css-ui/#cursor
+pub use values::computed::pointing::Cursor;
```

Comment:
Usually we do it the other way around (define it in `specified`, re-export it from `computed`).

It also feels a bit weird to have `ToCss` in a different place than `Parse` for the same type.

Sorry for not having caught this before, the patch looks great otherwise. If you don't want to address it is fine, can be a followup or something.

Reply:
No problem. No need to open another issue, I'll change it in this patch today! :)